app/tasks.py

```diff
@@ -445,6 +445,74 @@ def _should_notify(self, count_of_unreviewed_notes):
                 'unreviewed_notes_threshold')
 
 
+class ApiPersonPostProcessor(utils.BaseHandler):
+    """A class to do post-processing on Persons received through the API.
+
+    Currently, this just pulls photos from URLs and stores them. If any other
+    post-processing is ever required for Persons, it should also be done in this
+    task, to avoid race conditions.
+    """
+
+    ACTION = 'tasks/api_person_post_processor'
+
+    @classmethod
+    def enqueue(cls, repo, person_id, countdown=0):
+        task_name = '%s-api-person-post-processing-%s' % (
+            repo, int(time.time()*1000))
+        path = '/%s/tasks/api_person_post_processor' % repo
+        taskqueue.add(name=task_name, method='GET', url=path,
+                      queue_name='api-post-processing',
+                      params={'id': person_id}, countdown=countdown)
+
+    def get(self):
+        person = model.Person.get(self.repo, self.params.id)
+        if person is None:
```

Comment:
You're right, it's the same in this case (since `person` can't be the empty string, zero, or False here). Changed to `if not person` (same for notes below).